### ISSUE TITLE
Show index sizes

### ISSUE THREAD

**Issue by Bilge** [HeidiSQL/HeidiSQL#1500]
**Is your feature request related to a problem? Please describe.**
Not really.

**Describe the solution you'd like**
Display index sizes.

**Describe alternatives you've considered**
Manually querying the `index_length` column on `mysql.tables` (see https://dev.mysql.com/doc/refman/5.7/en/information-schema-tables-table.html).

**Comment by ansgarbecker** [comment]
What's wrong with the "Index length" column in the Database tab?

**Comment by Bilge** [comment]
I did not know of that since it is not shown by default and there is no clear indication additional information is hidden. However, having enabled it, what is wrong with it is that it's just the size of _all_ indexes for that table and does not show a breakdown of the size of each index for the table.

**Comment by ansgarbecker** [comment]
Is it possible to retrieve these single index sizes from the server?

**Comment by Bilge** [comment]
Yes, but apparently it is [storage-engine dependent](https://stackoverflow.com/questions/781873/how-to-figure-out-size-of-indexes-in-mysql/29443337).

**Comment by ansgarbecker** [comment]
Nice thing, works at least for the widely used InnoDB tables:

```sql
SELECT index_name, stat_value * @@innodb_page_size AS bytes
FROM mysql.innodb_index_stats
WHERE database_name='test9' AND table_name='test_table_zzz' AND stat_name='size' AND index_name='idx_actor_last_name'
```

I'll add this as a feature for the table editor's index editor.

**Comment by ansgarbecker** [comment]
Done for the next builds:

<img width="779" height="234" alt="Image" src="https://github.com/user-attachments/assets/0311bd1c-d95c-4b67-b709-ac2fb5a14c11" />
